pyproject.toml

```diff
@@ -89,8 +89,7 @@ ignore = [
     "RUF012", # Mutable class attributes should be annotated with `typing.ClassVar`
 ]
 
-# py36, but ruff does not support it :/
-target-version = "py37"
```

Comment:
I think the comments should be on the "minimum version" rather than "maximal supported version" (regardless of how those settings are named).

https://github.com/pylint-dev/pylint-pytest/actions/runs/6637112680/job/18031541929?pr=22